### ISSUE TITLE
Channels catalogues are missing

### ISSUE THREAD

**Issue by loglux** [yuliskov/SmartTube#1097]
When I open a channel in SmartTubeNext all videos are in one big mess and the channel's calalogization is missing. If the channel is big, it's not possible to locate a needed playlist, e.g. video lessons sorted by topics.

**Comment by costinmx** [comment]
If you open a Channel from the left sidebar menu - this is a subscribed Channel section, and you will see only latest uploads section from the channel. It not a fault, it's a YouTube style design feature. 
For full featured Channel - you should long press the OK key to activate the context menu, and select 'Open Channel', then you will get all the catalogues from this channel...

**Comment by loglux** [comment]
No. YouTube app is able to show the catalogue of the channel but SmartTubeNext doesn't show it.  If in YouTube app we can navigate through the channel's lists, SmartTubeNext shows one unsorted chaotic list of videos. If a channel is big enough it's not possible to locate a video you're looking for.

**Comment by costinmx** [comment]
Some similar problems are solved in v13.60, are you still experiencing issues?

**Comment by loglux** [comment]
I don't have further updates since v12.53.  Tried the manual update and tried to change the source from GitHub to google drive. Nothing happens.

**Comment by costinmx** [comment]
Can't imagine why manual update does not work for you. Looks like you just have updating issues, because you are using a long time outdated version. There are at least 3 or 4 Stable version updates since then, with a lot of improvements and fixes. The issue in this topic has been solved.

**Comment by loglux** [comment]
Removed my Beta and installed the latest release 13.60
There is still no possibility to see a sorted catalogue of subchannels (go to the channel option on the YT. when you can see all subchannels or sublists on the current channel and subscribe them)

**Comment by loglux** [comment]
To have this option forced to use the original YT app to locate and subscribe sublists

**Comment by costinmx** [comment]
> There is still no possibility to see a sorted catalogue of subchannels (go to the channel option on the YT. when you can see all subchannels or sublists on the current channel and subscribe them)

This is a different matter. The YT Apps don't have all the YT website features. And may never have it. 
I don't see any catalogue sorting options on YT original app.
The issue 'catalogue titles misssing' has been solved.

**Comment by loglux** [comment]
No, it doesn't work

The only way we can see a sorted catalogue on the channel if we open a video and click on the 'profile' icon (next to HQ icon). 

But we go to the channels and click on the channel we can see only an unsorted mess of all videos.

**Comment by costinmx** [comment]
My friend, this is the way YT works:
1st: 
'Channels' section sidebar, are all about **subscribed** channels. It's an YT feature, **just like** in original app. It's no mess at all, these are the latest videos from Upload section of each channel, **sorted** by newest video first. 
Go to YT app and in Subscriptions you will find the same list of subscribed channels, with 'uploads' section videos only. 

2nd: 
If you want to access all the categories, or catalogues, of a channel, you have to do it  **just like** in original app. As you described above. Long press on a card and selecting Open Channel.

This topic may be open forever, cause it's not a fault, it's a YT design feature.
